### PR TITLE
Update d/control

### PR DESCRIPTION
**Pull request by ahoneybun**
Add pop-launcher as a dep

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -12,6 +12,7 @@ Build-Depends:
   libxkbcommon-dev,
   libwayland-dev,
   pkg-config,
+  pop-launcher,
```
Comment (jackpot51) [inline debian/control:15]: Should this be a build depends?

Comment (wash2) [reply debian/control:15]: I feel like it could just be a depends instead

File: debian/control [hunk 2]
```diff
@@ -12,6 +12,8 @@ Build-Depends:
   libxkbcommon-dev,
   libwayland-dev,
   pkg-config,
+Depends:
```
Comment (jackpot51) [inline debian/control:15]: This needs to move down to the depends below that include shlibs.

Comment (ahoneybun) [reply debian/control]: Should be good now.

**Review by wash2 — APPROVED** [review]

**Pull request merged**